### PR TITLE
Feature/wanted outcome question

### LINKED ISSUE #31: wanted outcome - extra field
## Value proposition

As an User
I want a extra-input
So that I can write why I didn't reach my wanted outcome in an situation

## Description

Image

## Acceptance criteria

- [x] add the radio buttons and the functionality to the input forms "Vergangenheitsanalyse", Zukunftsanalyse" "Innere Situationsanalyse"
- [x] I want a Yes and No button/radio-Input on the question "Did I reach my wanted outcome?"
- [x] with a click on no i want a input-field where I can write the reason why the wanted outcome was not reached

## Tasks

- [x] have two `radio buttons`
- [x] let them say `"Ja"` and `"Nein"`
- [x] with a click on "Nein" an Input appears
- [x] make it `long` and `required`
- [x] `maxLength = 500`

### PR DESCRIPTION
**Pull request by vaupunkt**
# PR for [UserStory](https://github.com/vaupunkt/cbaspCompanion/issues/31)

As an User
I want a extra-input
So that I can write why I didn't reach my wanted outcome in an situation

### CODE REVIEW

File: components/AnalysisForm/index.js [hunk 1]
```diff
@@ -191,6 +202,51 @@ export default function AnalysisForm({ typeOfAnalysis }) {
...
+                  id="comparisonYes"
+                  name="comparison"
+                  value={true}
+                  checked={outcomeReached === true}
+                  onChange={onOptionChange}
+                />
+                <label htmlFor="comparisonYes">Ja</label>
+              </OutcomeReachedSection>
+              <OutcomeReachedSection>
+                {" "}
```
Comment (josch87) [inline components/AnalysisForm/index.js:224]: What do you need the space for?

File: components/AnalysisForm/index.js [hunk 2]
```diff
@@ -191,6 +202,51 @@ export default function AnalysisForm({ typeOfAnalysis }) {
             </>
           );
         }
+        if (analysisKey === "comparison") {
```
Comment (josch87) [inline components/AnalysisForm/index.js:205]: really good job with the analysisKeys!

**Review by josch87 — APPROVED** [review]
Looks very good!

File: components/AnalysisForm/index.js (hunk 1 above)
Comment (vaupunkt) [reply components/AnalysisForm/index.js:224]: oh, i think that was a mistake made by prettier :) 
thanks for seeing it!

**Pull request merged**